lib/dynosaur/process/heroku.rb

```diff
@@ -0,0 +1,42 @@
+require 'dynosaur/client/heroku_client'
+
+require_relative '../process'
+
+# Start a detached rake task on a one-off dyno
+module Dynosaur
+  class Process
+    class Heroku < Process
+      # Valid dyno sizes under Heroku's new pricing model
+      module Size
+        PERFORMANCE_M = 'performance-m'.freeze
+        PERFORMANCE_L = 'performance-l'.freeze
+        STANDARD_1X = 'standard-1X'.freeze
+        STANDARD_2X = 'standard-2X'.freeze
+      end
+
+      # Valid dyno sizes under Heroku's old pricing model
+      module DeprecatedSize
+        PERFORMANCE = 'PX'.freeze
+        STANDARD_1X = '1X'.freeze
+        STANDARD_2X = '2X'.freeze
+      end
+
+      def start
+        app_name = Dynosaur::Client::HerokuClient.app_name
+        dyno_accessor = Dynosaur::Client::HerokuClient.client.dyno
+        create_opts = { command: rake_command, attach: false }
```

Comment:
It'd be nice for Dynosaur to expand in to running arbitrary tasks. Another thing for later.